clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10136,6 +10136,9 @@ def err_vec_builtin_incompatible_vector : Error<
 def err_vsx_builtin_nonconstant_argument : Error<
   "argument %0 to %1 must be a 2-bit unsigned literal (i.e. 0, 1, 2 or 3)">;
 
+def err_vectorelements_non_vector : Error<
+ "'__builtin_vectorelements' argument must be a vector">;
```

Comment:
Do we use similar language elsewhere?  "Must be a vector" is pretty confusing.

Reply:
this was inspired by "`first argument to __builtin_convertvector must be a vector`". but if you have a suggestion, i'm happy to change this.

Reply:
Actually, we probably want to combine those two with a 'select' or insertion instead of keeping them as separate.  I might do something like `must be of vector type` instead of `a vector`, unless there is something more obviously specific (I think `SIMD vector type` would be incorrect, right?)